src/ai/ai_gui.cpp

```diff
@@ -190,7 +190,7 @@ struct AIConfigWindow : public Window {
 					TextColour tc = TC_SILVER;
 					if (this->selected_slot == i) {
 						tc = TC_WHITE;
-					} else if (IsEditable((CompanyID)i)) {
+					} else if (this->IsEditable((CompanyID)i)) {
```

Comment:
`IsEditable()` is a `static` method so we deliberately don't use `this->` for that.

Reply:
oops